doc/CommonTraps.md

```diff
@@ -34,3 +34,14 @@ This document details the common pitfalls when developing for PowerPointLabs, so
 1. Event triggering order of `WindowDeactivated` and `PresentationClose` is different between PowerPoint 2010 and 2013. In 2010, `PresentationClose` will be triggered before `WindowDeactivated`, but this order is reversed in 2013
 
 1. Accessing the `ActivePresentation.Saved` property on a new, unsaved presentation, always results in the `ActivePresentation.Saved` property returning `msoFalse`, regardless of whether the new presentation contains any valid changes to be saved.
+
+1. Visual Studio may show the error "Output Type of Class Library cannot be started directly", shown below. This is caused by errors in Visual Studio's project configuration.
+
+![Alt text](../doc/images/CommonTraps/output-type-class-library.png)
```

Comment:
The image and following text don't seem to line up well. Could you add tabs like in item 12 of the list?

![image](https://user-images.githubusercontent.com/11351531/50080057-343df800-0226-11e9-99d2-6ba1c89f20c5.png)